src/models/device-type.ts

```diff
@@ -13,7 +13,22 @@ const baseSchema = Joi.object().keys({
     pinFieldConfigurations: fieldConfigurationsFromServerSchema.required()
       .description('Defines deviceFields on the pin the channel is connected to. Can be used in report type functions. See the chapter on open fields on how to use this'),
     defaultPinName: Joi.string().example('Anode').description('If undefined, the channel cannot be linked to a pin'),
+    charts: Joi.array().items(Joi.object().keys({
```

Comment:
As we'll only implement this client side after the server implements this change: shall we make both `models.deviceType.charts` and `models.deviceType.channels[].charts` required and non-null? Saves the front-end the hassle of checking that every time